src/hooks/useGetContractState.tsx

```diff
@@ -46,21 +46,22 @@ const useGetContractState = (selectedAccountType: string) => {
         );
         const getContactState = await plrStakingGetContract?.callGetContractState();
         const staked = await plrStakingGetContract?.callGetStakedAmountForAccount?.(
-          selectedAccountType === AccountTypes.Key ? providerAddress : accountAddress
+          selectedAccountType === AccountTypes.Contract ? accountAddress : providerAddress
```

Comment:
The first thing is this is not only for Pillar DAO.
Only DAO uses Key Wallet. others who use this builder may use both

selectedAccountType === AccountTypes.Contract ? accountAddress : providerAddress
-> For Pillar DAO selectedAccountType is by default Key. And default Contract for those who use both (for both user can change account type)
In short, Key for simple dashboard otherwise Contract